src/components/scene/postprocessing.js

```diff
@@ -0,0 +1,5 @@
+var register = require('../../core/component').registerComponent;
```

Comment:
Can we keep the `tock` related code and move the postprocessing component to a different PR?